website/docs/developer/research/research-logging-libraries.md

```diff
@@ -116,6 +118,13 @@ Resulting Cloudwatch message:
 ```
 
 ## Recommendation
+- As the [Python CumulusLogger](https://github.com/nasa/cumulus-message-adapter-python#logging-with-cumuluslogger) is tied to Cumulus, 
+  it is recommended that we decouple from it.
+  - Any references to deployed layer should removed from Terraform.
+  - Schema validation checks should be added to calling function.
+  - References to run_cumulus_task should be replaced with manual transformation of inputs into non-architecture-specific formats.
```

Comment:
We should also create a card for this task. 

Seems like additional work if we decide to move away from run_cumulus_task

Reply:
It won't be much, actually. This list is the entirety of necessary changes.
Schema validation will be covered by upcoming pydantic changes, and we don't use the more complex parts of input parsing. For minimum-viable-product it'd look like `input = event["input"] config = event["config"] task(input, config)

Reply:
I'll create cards once plan sounds good to everyone.

Reply:
sounds good